app/views/unattended/provisioning_templates/PXEGrub/pxegrub_default_local_boot.erb

```diff
@@ -4,8 +4,12 @@ name: PXEGrub default local boot
 model: ProvisioningTemplate
 %>
 <%# Used to boot provisioned hosts, do not associate or change the name. %>
-
-default=0
+<%
+  # Grub1 only supports numeric default statement, this array allows conversion to number
+  items = ["local", "local_chain_hd0"]
+  default_item = items.index(global_setting("default_pxe_item_global", "local")) || 0
+%>
+default=<%= default_item %>
```

Comment:
We have this in global template, I realized we can use the same in local template. The code is the same and it has been tested already.